include/picongpu/simulation/stage/AtomicPhysics.hpp

```diff
@@ -302,8 +303,9 @@ namespace picongpu::simulation::stage
                         particles::atomicPhysics::stage::CalculateStepLength<boost::mpl::_1>>;
                     ForEachIonSpeciesCalculateStepLength{}(mappingDesc);
 
-                    // reject overSubscription loop, ends when no histogram bin oversubscribed
-                    while(true)
+                    // choose transition loop
+                    bool histogramOverSubscribed = true;
```

Comment:
later you have a variable named `noHistogramOverSubscribed`. The namings are confusing, they sounds euqal but most likely they are not.

Reply:
outdated, entire section has been refactored.